collectors/o365/lib/o365_mgmnt/o365management.js

```diff
@@ -262,52 +131,6 @@ class O365Management extends msRestAzure.AzureServiceClient {
         return client.sendRequest(httpRequest).then(handler);
     }
     
-    subscriptionsNotifications(contentType, startTs, endTs, options, callback) {
-        let client = this;
-        // Construct URL
-        let baseUrl = this.baseUri;
-        let tenantId = this.officeTenantId;
-        let publisherId = this.publisherId;
-        let requestUrl = baseUrl + (baseUrl.endsWith('/') ? '' : '/') +
-            'api/v1.0/' + encodeURIComponent(tenantId) +
-            '/activity/feed/subscriptions/notifications';
-        let queryParameters = [];
-        queryParameters.push('contentType=' + encodeURIComponent(contentType));
-        if (startTs && startTs !== '') queryParameters.push('startTime=' + encodeURIComponent(startTs));
-        if (endTs && endTs !== '') queryParameters.push('endTime=' + encodeURIComponent(endTs));
-        queryParameters.push('PublisherIdentifier=' + encodeURIComponent(publisherId));
-        if (queryParameters.length > 0) {
-            requestUrl += '?' + queryParameters.join('&');
-        }
-    
-        const httpOptions = {};
-        if (!this.generateClientRequestId) {
-            httpOptions.disableClientRequestId = true;
-        }
-
-        // Create HTTP transport objects
-        let httpRequest = new msWebResource(httpOptions);
-        httpRequest.method = 'GET';
-        httpRequest.url = requestUrl;
-        // Set Headers
-        if (this.acceptLanguage !== undefined && this.acceptLanguage !== null) {
-            httpRequest.headers.set('accept-language', this.acceptLanguage);
-        }
-        if(options) {
-            for(let headerName in options.customHeaders) {
-                if (options.customHeaders.hasOwnProperty(headerName)) {
-                    httpRequest.headers.set(headerName, options.customHeaders[headerName]);
-                }
-            }
-        }
-        httpRequest.headers.set('Content-Type', 'application/json; charset=utf-8');
-        httpRequest.body = null;
-        // Create handler
-        const handler = this.requestHandler(httpRequest);
-        // Send Request
-        return client.sendRequest(httpRequest).then(handler);
```

Comment:
Is all of this from the old SDK and that's why it's being nuked?

Reply:
yeah. It was from the old collector and not being used. Didnt feel like it was worth writing tests for it!